recipe/meta.yaml

```diff
@@ -21,15 +21,16 @@ requirements:
   run:
     - python
     - libconda
+    - pyyaml
+    - pycosat >=0.6.1
     - pillow >=3.1  # [win]
-    - nsis          # [win]
+    - nsis >=3.01   # [win]
 
 test:
   imports:
     - constructor
-
   commands:
-    - constructor --help
+    - constructor --test
```

Comment:
Why not `--help`?

Reply:
Guess [`AnacondaRecipes` does the same thing]( https://github.com/AnacondaRecipes/constructor-recipe/blob/85e1945d32f349d9b3c10642527a4ec71e0b3c48/recipe/meta.yaml#L25 ). Still curious as to why you made the switch, but it is non-blocking.

Reply:
Well, I guess you can blame it on my general packaging paranoia. ;-)
But I feel that even though the release itself should be properly tested, why not run tests if possible during packaging as well? There's always something can go wrong during packaging, even if the release is looking good.